compiler+runtime/src/cpp/main.cpp

```diff
@@ -101,7 +101,7 @@ namespace jank
     using namespace jank;
     using namespace jank::runtime;
 
-    //__rt_ctx->load_module("/clojure.core", module::origin::latest).expect_ok();
+    __rt_ctx->load_module("/clojure.core", module::origin::latest).expect_ok();
```

Comment:
We can't just uncomment this unconditionally yet. This is why you saw the duplicate symbol error when compiling clojure.core.

Reply:
Without it, compiling any module fails. The compiler can't recognize any clojure.core symbol used in the module being compiled.

Reply:
Yep, that makes sense. We need to add a special case for compiling `clojure.core` then, to not also load `/clojure.core`. For now, this works around a bug (the duplicate symbols), but it also makes sense just from an efficiency standpoint.